Header header
float32 value
geometry_msgs/Point position
string type